### PR TITLE
Changing artifact id

### PR DESCRIPTION
**Pull request by andresguedes**
This PR changes the Artifact ID, that needs to be the same as the project path's name, to build and upload the .AAR as expected

### CODE REVIEW

**Review by marcosrivereto — APPROVED** [review]
👍

File: trustly-android-sdk/build.gradle [hunk 1]
```diff
@@ -66,7 +66,7 @@ apply from: 'publish-root.gradle'
 ext {
     PUBLISH_GROUP_ID = 'net.trustly'
     PUBLISH_VERSION = '3.0.0'
-    PUBLISH_ARTIFACT_ID = 'android-sdk'
+    PUBLISH_ARTIFACT_ID = 'trustly-android-sdk'
```
Comment (lukevance) [inline trustly-android-sdk/build.gradle:69]: Will this be used anywhere visible to a developer user? Just wondering if it should be the same as the repo name or if this is ok. If it's not going to be used as the package name or anything, this is fine 👍

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ it, include the dependency below in your `build.gradle` file.
 
 ```
 dependencies {
-    implementation 'net.trustly:android-sdk:3.0.0'
+    implementation 'net.trustly:trustly-android-sdk:3.0.0'
```
Comment (lukevance) [inline README.md:32]: 🕶️

**Review by lukevance — COMMENTED** [review]
Looks good, just the one question.

File: trustly-android-sdk/build.gradle (hunk 1 above)
Comment (andresguedes) [reply trustly-android-sdk/build.gradle:69]: We thought that this is an internal reference there in Maven Central, and it should be the same as the package folder or project name

**Pull request merged**